### PR TITLE
moves breadcrumbs to common-ngx

### PR DESCRIPTION
**Pull request by esotrope**
With this change admin and reporting breadcrumbs will behave identically.

### CODE REVIEW

File: common-ngx/src/layout/rdw-layout.module.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    BrowserModule,
+    FormsModule,
+    RouterModule.forRoot([]),
+    TranslateModule.forRoot()
+  ],
+  exports: [
+    Breadcrumbs
+  ]
+})
+export class RdwLayoutModule {
```
Comment (esotrope) [inline common-ngx/src/layout/rdw-layout.module.ts]: was thinking this package could hold the nav bar and footer as well - open to suggestions on a better name

File: webapp/src/main/webapp/src/app/breadcrumbs/breadcrumbs.component.ts [hunk 2]
```diff
@@ -1,112 +0,0 @@
-import { Component, OnInit } from "@angular/core";
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/breadcrumbs/breadcrumbs.component.ts]: @wtritch some more lovely red for you

File: common-ngx/src/layout/breadcrumbs.component.ts [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+      return this.createBreadcrumbs(child, routerLinkParameters, breadcrumbs);
+    }
+  }
+
+  private createBreadcrumb(options: BreadcrumbOptions, routeData: any, routerLinkParameters: any[]): Breadcrumb {
+    if (options.translate) {
+      let text = options.translateResolve
+        ? this.translateService.instant(options.translate, Utils.getPropertyValue(options.translateResolve, routeData))
+        : this.translateService.instant(options.translate);
```
Comment (esotrope) [inline common-ngx/src/layout/breadcrumbs.component.ts]: Now translating this when creating the crumb at ngOnInit time. Before we did this twice in the create title method and in the template.
This makes it to where we don't have to pass translateParams & requiresTranslate props around.

File: common-ngx/src/layout/rdw-layout.module.ts (hunk 1 above)
Comment (wtritch) [reply common-ngx/src/layout/rdw-layout.module.ts]: Hm...  It'll hold basic page structure/framing components... UIStructureModule? ...just shooting in the dark.

**Pull request merged**